feat: Adds localization column in the alerts table

blenzi opened the pull request:
Following (#267), this PR adds a localization field in the alerts table, including minimal necessary changes.
The localization field is meant to be used as a JSON dump of a list of bounding boxes.

MateoLostanlen reviewed (commented):
Hi, thanks for the PR, it might be nice to have a test with a different localization than None.

MateoLostanlen commented on Makefile:
```diff
@@ -17,7 +17,7 @@ lock:
 build:
 	poetry export -f requirements.txt --without-hashes --output src/app/requirements.txt
-	docker build src/. -t pyronear/pyro-api:python3.8-alpine3.10
+	docker build src/. -t pyronear/pyro-api:python3.8-alpine3.10-prod
```
I might not be the right PR to change this

blenzi replied:
Thanks for spotting. I copied it from the previous one. Removed now

blenzi commented:
Thanks @MateoLostanlen . There is a value that is not None but it was not according to the desired format (`conf` missing). I added some validation checks and fixed it

frgfm commented:
Closing this since it has been merged in #267
